src/System.Security.Cryptography.Cng/tests/System.Security.Cryptography.Cng.Tests.csproj

```diff
@@ -101,6 +102,7 @@
     <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\ECDsa\ECDsaTestData.cs">
       <Link>CommonTest\System\Security\Cryptography\AlgorithmImplementations\ECDsa\ECDsaTestData.cs</Link>
     </Compile>
+    <Compile Include="RSACngProvider.cs" />
```

Comment:
Curious: Were these tool-changed or hand-reordered?  The previous ordering had all local files together, then all imported files.  Now it's a bit harder to see the local files.